optimize.sh

```diff
@@ -12,6 +12,7 @@ echo "Info: sccache stats before build"
 sccache -s
 
 mkdir -p artifacts
+true >artifacts/checksums_intermediate.txt
```

Comment:
What is this supposed to do? Why not

```suggestion
touch artifacts/checksums_intermediate.txt
```

Reply:
No, this is not the same. Committed in haste. Will revert. What I want is to make sure the file doesn't exist / is empty.

maybe `rm -f` is clearer?

Reply:
`echo -n "" > artifacts/checksums_intermediate.txt` would be empty+existing
`rm -f` is you want non-existent

Reply:
Yes. I wasn't sure `tee -a` would work without a file, that's why i used that idiom.

Reply:
I really like `$ >file` idiom, by the way. It means: "Doing nothing produces nothing. And, redirecting nothing to a file, produces an empty file."